src/interpreter/ByteCodeGenerator.cpp

```diff
@@ -611,7 +617,8 @@ ByteCodeBlock* ByteCodeGenerator::generateByteCode(Context* c, InterpretedCodeBl
             case BinarySignedRightShiftOpcode:
             case BinaryUnsignedRightShiftOpcode:
             case BinaryInOperationOpcode:
-            case BinaryInstanceOfOperationOpcode: {
+            case BinaryInstanceOfOperationOpcode:
+            case BinaryExponentiationOpcode: {
```

Comment:
`BinaryExpressionNullishCoalescing` bytecode does not need stack index alignment?